### PR TITLE
added many to many relationship between user and clients

### PR DESCRIPTION
**Pull request by Dhroov7**
Done #414

### CODE REVIEW

File: src/db/models.js [hunk 1]
```diff
@@ -193,7 +204,7 @@ module.exports = {
     models: {
         User, UserLocal, UserFacebook, UserTwitter, UserGithub, UserGoogle,
         UserLinkedin, UserLms, Client, GrantCode, AuthToken, Resetpassword, Verifyemail,
-        Demographic, Address, College, Company, Branch, State, Country, EventSubscription
+        Demographic, Address, College, Company, Branch, State, Country, EventSubscription, userClient
```
Comment (championswimmer) [inline src/db/models.js]: Why name this model with small camelcase when every other is capital camelcase

Comment (Dhroov7) [reply src/db/models.js]: @championswimmer Done!

**Comment by Dhroov7** [comment]
screenshot 2018-11-24 at 2 55 58 pm
screenshot 2018-11-24 at 3 28 44 pm


@championswimmer Please review the Design.

**Comment by Dhroov7** [comment]
@championswimmer Done with owner addition feature..Please review.

**Pull request closed without merging**